Event/Event.gd

```diff
@@ -0,0 +1,11 @@
+extends Resource
+class_name Event
```

Comment:
would rename to `EventBase` (because even if we didn't agree explicitly on that, it seems to be the convention for parent classes until now)